jbpm-designer-backend/src/main/java/org/jbpm/designer/filter/InjectionConfig.java

```diff
@@ -37,38 +38,38 @@ public InjectionConfig(ServletContext sc) {
     }
 
     public synchronized void load(String contextPath) {
-        InputStream is = null;
-        is = this.getClass().getResourceAsStream(DEFAULT_INJECTION_CONF_FILE);
-        if (is == null) {
-            System.out.println("unable to find designer injection conf file "
-                    + DEFAULT_INJECTION_CONF_FILE);
-        }
-
         DocumentBuilder parser;
         DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
         factory.setValidating(false);
         factory.setNamespaceAware(false);
-        // factory.setIgnoringComments(true);
-        // factory.setIgnoringElementContentWhitespace(true);
         try {
             parser = factory.newDocumentBuilder();
         } catch (ParserConfigurationException e) {
             // TODO log
             return;
         }
+        InputStream is = null;
         try {
-            Document doc = parser.parse(is);
-            NodeList rulesConf = doc.getElementsByTagName("rule");
-            rules = new InjectionRules();
-            for (int i = 0; i < rulesConf.getLength(); i++) {
-                Node ruleNode = rulesConf.item(i);
-                InjectionRule rule = new InjectionRule(ruleNode, contextPath);
-                if (rule.isEnabled())
-                    rules.add(rule);
+            is = this.getClass().getResourceAsStream(DEFAULT_INJECTION_CONF_FILE);
+            if (is == null) {
+                System.out.println("unable to find designer injection conf file "
+                        + DEFAULT_INJECTION_CONF_FILE);
+            } else {
+                Document doc = parser.parse(is);
+                NodeList rulesConf = doc.getElementsByTagName("rule");
+                rules = new InjectionRules();
+                for (int i = 0; i < rulesConf.getLength(); i++) {
+                    Node ruleNode = rulesConf.item(i);
+                    InjectionRule rule = new InjectionRule(ruleNode, contextPath);
+                    if (rule.isEnabled())
+                        rules.add(rule);
+                }
             }
         } catch (Exception e) {
             // TODO Log
             e.printStackTrace();
```

Comment:
@tsurdilo Hi Tihomir. How do you think is it good enough to use org.slf4j.Logger here? It will be better to use Logger#info or Logger#error?